src/sentry/south_migrations/0286_drop_project_fk_release_release_file.py

```diff
@@ -0,0 +1,730 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+from sentry.utils.db import is_mysql, is_postgres, is_sqlite
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        if is_postgres():
+            # Changing field 'Release.project_id'
+            db.execute("ALTER TABLE sentry_release DROP CONSTRAINT project_id_refs_id_21d237e2")
+
+            # Changing field 'ReleaseFile.project_id'
+            db.execute("ALTER TABLE sentry_releasefile DROP CONSTRAINT project_id_refs_id_878696ea")
+        elif is_mysql():
```

Comment:
From testing locally, it appears that the `project_id` FK constraints don't exist, so we can ignore anything for MySQL. I confirmed this on both `sentry_release` and `sentry_releasefile`.